pkgs/development/python-modules/yaspin/default.nix

```diff
@@ -30,13 +31,13 @@ buildPythonPackage rec {
   ];
 
   nativeCheckInputs = [
+    pytest-xdist
     pytestCheckHook
   ];
 
-  postPatch = ''
-    # https://github.com/pavdmyt/yaspin/pull/212
-    substituteInPlace pyproject.toml \
-      --replace 'termcolor-whl = "1.1.2"' 'termcolor = "*"'
+  preCheck = ''
+    # tests assert for \033 which gets turned off in termcolor when TERM dumb is used which is used by nix
```

Comment:
let's avoid putting comments in nix strings. they unnecessarily affect drv hashes

Reply:
If you want to but that also doesn't really matter, not like we are going to update it anytime soon.